frdrpc/node_audit.go

```diff
@@ -53,9 +55,15 @@ func parseNodeAuditRequest(ctx context.Context, cfg *Config,
 		return nil, nil, err
 	}
 
+	pricePoints, err := pricePointsFromRPC(req.CustomPrices)
```

Comment:
Probably also want to only accept custom prices if `CUSTOM` was set?

Reply:
do you mean like error out if they provide prices but dont use custom option?

Reply:
yep! rather be really strict with what we're accepting so there's no confusion :)